host-shared/PKCS11Path.cpp

```diff
@@ -104,6 +104,7 @@ PKCS11Path::Params PKCS11Path::getPkcs11ModulePath() {
         WideCharToMultiByte(CP_UTF8, 0, path.c_str(), DWORD(path.size()), &result[0], size, nullptr, nullptr);
         return result;
     }();
+	static const std::string eTokenPath("C:\\Windows\\System32\\eTPKCS11.dll");
```

Comment:
This is unused on windows

Reply:
You're right, I'll remove it. However, I'm planning to use it conditionally for multi signing if I can not succeed on the native token manager of SafeNet in Windows.